examples/llm_text_generation/simple_model_training.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
+
```

Comment:
Should we have an equivalent of this file for llm models

Reply:
@skanjila I don't think I understand what you mean. Isn't this a file for LLM models? 
We don't support training an LLM from scratch yet, just the ability to use a pretrained LLM for text generation, zero shot learning and few shot learning (which should land in the next day). 

These example notebooks should be good enough for users to learn how to use Ludwig for these kinds of tasks